### PR TITLE
docs: add mfa login docs (u2f, totp, sms, email)

### PR DESCRIPTION
**Pull request by hifabienne**
Extend login ui guides with MFA authentication section. (U2F, TOTP)
Extend login ui guides with MFA registration and authentication for Email and SMS Code

### Definition of Ready

- [ ] I am happy with the code
- [ ] Short description of the feature/issue is added in the pr description
- [ ] PR is linked to the corresponding user story
- [ ] Acceptance criteria are met
- [ ] All open todos and follow ups are defined in a new ticket and justified
- [ ] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [ ] No debug or dead code
- [ ] My code has no repetitions
- [ ] Critical parts are tested automatically
- [ ] Where possible E2E tests are implemented
- [ ] Documentation/examples are up-to-date
- [ ] All non-functional requirements are met
- [ ] Functionality of the acceptance criteria is checked manually on the dev system.

### CODE REVIEW

File: docs/docs/guides/integrate/login-ui/mfa.mdx [hunk 1]
```diff
@@ -181,11 +471,73 @@ curl --request POST \
...
+
+![Authenticate U2F](/img/guides/login-ui/authenticate-u2f-flow.png)
+
+### Check User
+
+To be able to check the Universal-Second-Factor (U2F) you need a user check and a webAuthN challenge.
+In the creat session request you can check for the user and directly initiate the webAuthN challenge.
+
+For U2F you can choose between "USER_VERIFICATION_REQUIREMENT_PREFERRED" and "USER_VERIFICATION_REQUIREMENT_DISCOURAGED" for the challenge.
+Best practice is using discouraged, as this doesn't require the user to enter a pin. With preferred the PIN will be shown, but is not necessary.
```
Comment (livio-a) [inline docs/docs/guides/integrate/login-ui/mfa.mdx:495]:
```suggestion
Best practice is using discouraged, as this doesn't require the user to enter a PIN. With `preferred` the user might be prompted for the PIN, but is not necessary.
```

File: docs/docs/guides/integrate/login-ui/_browser_signin_webauthn.mdx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+
+After starting the passkey authentication on the side of ZITADEL you have to challenge the browser.
```
Comment (livio-a) [inline docs/docs/guides/integrate/login-ui/_browser_signin_webauthn.mdx:2]:
```suggestion
After starting the WebAuthN authentication on the side of ZITADEL you have to challenge the browser.
```

Comment (livio-a) [reply docs/docs/guides/integrate/login-ui/_browser_signin_webauthn.mdx:2]: as this is now used for passkeys and U2F

**Review by livio-a — CHANGES_REQUESTED** [review]

File: docs/docs/concepts/features/selfservice.md [hunk 3]
```diff
@@ -152,6 +152,8 @@ Users can setup and delete a second factor and FIDO Passkeys (Passwordless).
 Available authenticators are:
 
 - Mobile one-time password (OTP) (Authenticator Apps)
+- One time E-Mail Code
+- One time SMS Code
```
[the remark this answers is not in the document]
Comment (hifabienne) [reply docs/docs/concepts/features/selfservice.md:156]: yes makes sense, will change. I just forgot again about it

**Review by livio-a — APPROVED** [review]

**Pull request merged**